Neos.Neos/Classes/ViewHelpers/Backend/IfModuleAccessibleViewHelper.php

```diff
@@ -82,6 +82,7 @@ protected static function evaluateCondition($arguments, RenderingContextInterfac
         if (!$privilegeManager->isGranted(ModulePrivilege::class, new ModulePrivilegeSubject($arguments['modulePath']))) {
             return false;
         }
+        /** @phpstan-ignore-next-line condition will be fixed with Neos 9 */
```

Comment:
it _should_ be `$arguments['moduleConfiguration']['privilegeTarget']` but i dont know what the consequences of changing this now are. Also this is fluid so

Reply:
Ihhh. And $moduleConfiguration doesn't even exist 🙈